test/auto_parallel/hybrid_strategy/parallel_api.py

```diff
@@ -196,6 +201,18 @@ def check_mp(self, layer):
                     ]
 
     def parallel_model(self, layer, optimizer=None):
+        if self.pp > 1:
+            decoders_per_rank = self.config.num_hidden_layers // self.pp
+            split_spec = OrderedDict(
+                [
+                    (
+                        f"llama.layers.{i * decoders_per_rank - 1}",
+                        SplitPoint.END,
+                    )
+                    for i in range(1, self.pp)
+                ]
+            )
```

Comment:
may be it's better to use tuple of list, we can discuss it later.

Reply:
OK